crates/rpc-types-trace/src/geth/call.rs

```diff
@@ -56,6 +56,9 @@ pub struct CallLogFrame {
     /// The data of the log.
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub data: Option<Bytes>,
+    /// The position of the log relative to subcalls within the same trace.
+    #[serde(default, skip_serializing_if = "Option::is_none")]
```

Comment:
pls use alloy_serde::quantity::opt with u64 instead of ruint

Reply:
fixed, ptal